velox/functions/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ velox_add_library(velox_coverage_util CoverageUtil.cpp)
 
 velox_link_libraries(
   velox_function_registry
+  velox_exec
```

Comment:
I feel that generally we should avoid a dependency from velox/functions to velox/exec. Though I do see other places that have done so as well. 

@Yuhta : Do you have any strong feelings about this ?

Reply:
I would say let's keep it decoupled as much as we can.  Seems there is no real dependency of `WindowFunction` that need velox_exec, can we move that file here instead?

Reply:
@Yuhta : WindowFunction needs WindowPartition to access column values data in functions. WindowPartition needs RowContainer (which is in velox/exec).